README.md: add a section about adding "registrations" (#212)
osbuild/image-builder-cli: README.md

...
```

### Adding registrations

Adding registrations/subscriptions to an image can be done at
build time via the `--registrations` commandline option. When

> using this optionthe resulting image will include the given

Reviewer: ```suggestion
using this option the resulting image will include the given
```

subscriptions/registrations in the resulting image.

> Currently the RedHat subscription is supported, e.g.

Reviewer: ```suggestion
Currently the Red Hat subscription is supported, e.g.
```

...
```

### Adding registrations

Adding registrations/subscriptions to an image can be done at
build time via the `--registrations` commandline option. When

> using this option the resulting image will include the given1

Reviewer: ```suggestion
using this option the resulting image will include the given
```
